toolchain.mk

```diff
@@ -23,10 +23,13 @@ SRC_AARCH64_GCC 		?= https://developer.arm.com/-/media/Files/downloads/gnu-a/10.
 # $(3) is $.._GCC_VERSION	: the name of the file to download
 define dltc
 	@if [ ! -d "$(1)" ]; then \
-		mkdir -p $(1); \
 		echo "Downloading $(3) ..."; \
-		curl -s -L $(2) -o $(TOOLCHAIN_ROOT)/$(3).tar.xz; \
-		tar xf $(TOOLCHAIN_ROOT)/$(3).tar.xz -C $(1) --strip-components=1; \
+		curl --retry-all-errors --retry 5 -s -L $(2) -o $(TOOLCHAIN_ROOT)/$(3).tar.xz || \
```

Comment:
- Why `--retry-all-errors`? This is nor recommended in the `curl` man page and will for instance retry download when the local output directory doesn't exist (which is quite useless)
- Please add `-S` so that curl prints error an message on failure.

Reply:
Done.